querybook/server/lib/export/base_exporter.py

```diff
@@ -86,6 +89,29 @@ def _get_statement_execution_columns_len(
         return 0
 
     def _get_statement_execution_result(
+        self,
+        statement_execution_id: int,
+        raw: bool = False,  # If raw, return unparsed csv text
+        number_of_lines: int = 2001,
+    ):
+        """Warning: This function will be deprecated shortly as we move towards exporting full
```

Comment:
do something like `LOG.error('_get_statement_execution_result will be deprecated soon due to ...')`